### ISSUE TITLE
[monitoring] error message in alert should contain more info

### ISSUE THREAD

**Issue by cezio** [geosolutions-it/geonode#243]
* [x] period for alert occurrence
 * [x] units for threshold/offending values
 * [x] description of metric

**Comment by cezio** [comment]
Alert may be generated in different moment than value that was failing. 

I've added two timestamps to alert messages (send by email):
 * spotted (time when error was detected)
 * emerged (time when error value was recorded in db)

Email message contents:

> <p>Hi, admin</p>
> 
> <p>Monitoring on http://localhost:8000/ has detected following problems :</p>
> <ul>
>    <li><p> Number of handled requests is lower than 30
>       <p> spotted Sept. 4, 2017 12:55 p.m. - 0 minutes ago</p>
>       <p> emerged Sept. 4, 2017 12:54 p.m. - 1 minute ago</p>
>       <p>&nbsp;&nbsp;&nbsp; <small>Number of requests should be at least 30, got 7 instead</small></p></li>
> </ul>


Status API response also contains those timestamps:

```
{
  "status": "ok",
  "data": {
    "problems": [
      {
        "threshold_value": "30.0000",
        "valid_from": "2017-09-04T12:53:00",
        "name": "request.count",
        "check_url": "/monitoring/api/notifications/config/2/",
        "description": "Number of requests should be at least 30, got 7 instead",
        "valid_to": "2017-09-04T12:54:00",
        "check_id": 2,
        "spotted_at": "2017-09-04T12:57:18.863",
        "offending_value": "7.0000",
        "message": "Number of handled requests is lower than 30",
        "severity": "error"
      }
    ],
    "health_level": "error"
  },
  "success": true
}
```

Additionally, i've added `description` field to `monitoring.Metric`, so `request.count` becomes `Number of requests`. There's also `unit` field, exposed in API. It will be used in descriptions for non-count metrics (only in metrics that use physical units).


Sample email with units

<p>Hi, admin</p>

> <p>Monitoring on http://localhost:8000/ has detected following problems :</p>
> <ul>
>    <li><p> Incoming traffic should be higher than 10000000  B/s
>        <p> spotted Sept. 4, 2017 1:13 p.m. - 0 minutes ago</p>
>       <p> emerged Sept. 4, 2017 1:14 p.m. - 0 minutes ago</p>
>        <p>&nbsp;&nbsp;&nbsp; <small>for localhost-hostgeonode service Network incoming traffic rate should be at least 10000000 B/s, got 9457 B/s instead</small></p></li>
> </ul>

And API payload:

```
{
  "status": "ok",
  "data": {
    "problems": [
      {
        "threshold_value": "10000000.0000",
        "valid_from": "2017-09-04T13:10:00",
        "name": "network.in.rate",
        "check_url": "/monitoring/api/notifications/config/2/",
        "description": "for localhost-hostgeonode service Network incoming traffic rate should be at least 10000000 B/s, got 12019 B/s instead",
        "valid_to": "2017-09-04T13:11:00",
        "check_id": 2,
        "spotted_at": "2017-09-04T13:11:08.610",
        "offending_value": "12018.5417",
        "message": "Incoming traffic should be higher than 10000000 B/s",
        "severity": "error"
      }
    ],
    "health_level": "error"
  },
  "success": true
}
```

Basically, this depends much on metric definitions (good `description` values describing metric, properly set units and types of metric) and notification definitions. 
good `description` values for notification definition should obey template: *INDICATOR should be higher/lower than*. Then, threshold value and unit will be added in final message.
